gtests/net/mptcp/unsupported/test_client_1.pkt

```diff
@@ -0,0 +1,30 @@
+// connect() function, connection initiated by the kernel
+--tolerance_usecs=100000
```

Comment:
regarding the file name, certainly better to name it something like `ip_tos_client.pkt`